src/inverseRobot.js

```diff
@@ -24,7 +24,22 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+  const check = Object.values(robot);
+
+  check.sort();
+
+  for (let i = 0; i < check.length; i++) {
+    if (check[i] === check[i + 1]) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
+    result[robot[key]] = key;
```

Comment:
```suggestion
    if (robot[key] in result) {
      return null;
    }
    result[robot[key]] = key;
```
This approach is way more simple and understandable.